_plugins/content-modifier.rb

```diff
@@ -0,0 +1,98 @@
+# frozen_string_literal: true
```

Comment:
Does this need a `SPDX-License-Identifier` and a copyright?

Reply:
Added.